test/backend/PlotlyAPI.spec.js

```diff
@@ -24,31 +31,29 @@ describe('Grid API Functions', function () {
         // it works off of the assumption that a grid exists
 
         let fid;
-        return createGrid('Test updateGrid').then(res => res.json().then(json => {
-            // Update the grid's data
+        return createGrid('Test updateGrid')
+        .then(assertResponseStatus(201))
+        .then(getResponseJson).then(json => {
             fid = json.file.fid;
             const uids = json.file.cols.map(col => col.uid);
-
-            return wait(1000).then(() => {
-                return updateGrid(
-                    [
-                        ['x', 10, 40, 70, 100, 130],
-                        ['y', 20, 50, 80, 110, 140],
-                        ['z', 30, 60, 90, 120, 150]
-                    ],
-                    fid,
-                    uids,
-                    username
-                );
-            });
-        })).then(() => {
-            // Retrieve the contents from the grid
-            return wait(1000).then(() => {
-                const url = `grids/${fid}/content`;
-                return PlotlyAPIRequest(url, {username, apiKey, method: 'GET'});
-            });
-        }).then(res => res.json().then(json => {
-            // Test that the update worked
+            return updateGrid(
+                [
+                    ['x', 10, 40, 70, 100, 130],
+                    ['y', 20, 50, 80, 110, 140],
+                    ['z', 30, 60, 90, 120, 150]
+                ],
+                fid,
+                uids,
+                username
+            );
+        })
+        .then(assertResponseStatus(200))
```

Comment:
This may be a small thing.  But wondering if we should use a constant here

Reply:
I wouldn't define constants for this. Typically, I'd define a constant to make the code more readable. In this case, the function name `assertResponseStatus(200)` is enough to tell the reader 200 is a status code.